src/app/app.css

```diff
@@ -194,16 +194,25 @@ html, body, #root {
   --pf-c-label__content--before--BorderWidth: 2.5px;
 }
 
+:where(.pf-theme-dark) .recording-filter__toolbar-filter .pf-c-label {
+  --pf-c-label--BackgroundColor: var(--pf-global--BackgroundColor--400);
+}
+
 .pf-c-label-group.pf-m-vertical.pf-c-label-group {
   height: 100%;
   width: 100%;
-  background-color: var(--pf-global--Color--light-200);
-  border-color: var(--pf-global--BorderColor--100);
+  background-color: var(--pf-global--BackgroundColor--light-300);
+  border-color: var(--pf-global--BorderColor--light-100);
   border-style: solid;
   border-width: 0.1em;
   border-radius: 0.5em;
 }
 
+.pf-theme-dark .pf-c-label-group.pf-m-vertical.pf-c-label-group {
```

Comment:
I think we can remove this? There is already some styling for dark-mode label-group that looks quite the same?